docs/getting-started.md

```diff
@@ -0,0 +1,533 @@
+# Getting Started with Tekton Pipelines
+
+This guide helps you get started with Tekton Pipelines by walking you through a simple `Hello World` tutorial. The tutorial shows you how to:
+
+- Create a `Task`	
+- Create a `Pipeline` containing your `Tasks`	
+- Use a `TaskRun` to instantiate and execute a `Task` outside of a `Pipeline`	
+- Use a `PipelineRun` to instantiate and run a `Pipeline` containing your `Tasks`
+
+## Before you begin
+
+Before you begin this tutorial, make sure you have [installed and configured](install.md)
+the latest release of Tekton on your Kubernetes cluster.
+
+### Install Minikube
+If you would like to complete this tutorial on your local workstation, you will need Minikube.
+Install [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) v1.50 or higher.
+
+### Install Tekton CLI
+This getting started guide uses the Tekton CLI, tkn, so make sure to install that on your local machine.
+Install [Tekton CLI](https://github.com/tektoncd/cli). 
+
+### Sign in to Docker Hub
+You will need an account on [Docker Hub](https://hub.docker.com).
+
+## Creating and running a `Task`
+
+A [`Task`](tasks.md) defines a series of `Steps` that run in a desired order and complete a set amount of build work. Every `Task` runs as a Pod on your Kubernetes cluster with each `Step` as its own container. For example, the following `Task` outputs "Hello World":
+
+```yaml
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: echo-hello-world
+spec:
+  steps:
+    - name: echo
+      image: ubuntu
+      command:
+        - echo
+      args:
+        - "Hello World"
+```
+To run this `Task`, instantiate it using a [`TaskRun`](taskruns.md):
+
+```yaml
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: echo-hello-world-task-run
+spec:
+  taskRef:
+    name: echo-hello-world
+```
+
+Apply your YAML files as follows:
+
+```bash
+kubectl apply -f <YOUR-TASK.yaml>
```

Comment:
It would be great if these `Tasks` were defined somewhere under the docs folder so the user can just run `kubectl apply -f` without having to copy, create local file, and paste. It's done in a number of locations throughout the tutorial, and it's an easy fix for an easier user experience.

Reply:
I 100% Agree. There is an issue where the katacoda is also have these files as well. 
Should I duplicate the yaml on katacoda repo and this repo?
Should there be yaml in the docs folder? 
I think it is a better idea to link the the yaml to the katacoda folder. However, I still need to make an issue about opening a katacoda repo under the tekton org.

Reply:
then again (kubectl apply -f https//github....) isnt really a better use experience either 🤔